sotodlib/coords/pmat.py

```diff
@@ -501,9 +544,21 @@ def _get_asm(self):
         return so3g.proj.Assembly.attach(self.sight, so3g_fp)
 
     def _prepare_map(self, map):
-        if self.tiled: return map.tiles
+        if self.tiled and self.pix_scheme == "rectpix": return map.tiles
         else:          return map
```

Comment:
Make this 4 lines.  If you hadn't broken the alignment with the `else:` line I might have let this one slide. :)

The one liner is acceptable... `return (map.tiles if ... else map)`